src/cf.data.pre

```diff
@@ -4685,6 +4685,13 @@ DOC_START
 	Format codes:
 
 		%	a literal % character
+
+		byte{value}	Adds a single byte with the given value (e.g., %byte{10}
+			adds an ASCII LF character a.k.a. "new line" or "\n"). The value
```

Comment:
Just to clarify: adding %byte{10} results in a genuine new line in access.log (i.e., not a couple of characters "\n"). Is this an intended/expected behavior?

Reply:
> Is this an intended/expected behavior?

Yes, it is. In fact, IIRC, adding new lines is the use case that prompted the addition of this feature. Nearly all other ASCII characters can probably be added (as those characters) without a new logformat %code.

Adding new lines is useful when, for example, access records go to a logging daemon that expects HTTP header-like record syntax.